app/channel-settings/client/stylesheets/channel-settings.css

```diff
@@ -131,3 +131,23 @@ html.rtl .flex-tab {
 		}
 	}
 }
+
+.rc-input__textarea {
```

Comment:
why not textarea.rc-input__element, and set only properties you need, instead of set all properties again

Reply:
I did this because there were a lot of selectors coming from various files, which were not working well when I changed the tag from `input` to `textarea`. Probably, input tag had been used a selector somewhere else, so it was conflicting when I had tried to follow your suggested method before.

Reply:
@ggazzo